framework/tests/accumulation/SimpleInferenceMerge.java

```diff
@@ -0,0 +1,37 @@
+import testaccumulation.qual.*;
+
+class SimpleInferenceMerge {
```

Comment:
This test case is unrelated to predicates, right?

Reply:
Yeah. It probably should have been included in the original AccumulationChecker PR, but was left off somehow. I included it here since it passes and I moved over a bunch of the other tests, so it didn't seem like it needed its own.